core/app/models/spree/exchange.rb

```diff
@@ -39,10 +40,6 @@ def self.param_key
       "spree_exchange"
     end
 
-    def self.model_name
-      Spree::Exchange
-    end
-
     def model_name
```

Comment:
I didn't have any tests fail when taking it out.  https://github.com/solidusio/solidus/pull/235, which brings it in, doesn't actually tell us why it's needed either.